### ISSUE TITLE
Gain very bad

### ISSUE THREAD

**Issue by F4HZA** [rfrht/FT991A-PAT#25]
hello, 

Thank you for this project, great work, I like the integration in the FT-991A.
I realized the S revision with the BRF193F, but I have a gain problem.
The gain is around -20db, I changed the BFR193F but the same.

I tested in the FT-991A but the attenuation is too high with IPO or AMP1 the signals are almost invisible with AMP2 I see the signals.

Do you have an idea of my problem? Is the new BFR193F transistor not suitable?

Thanks.
73.

**Comment by rfrht** [comment]
Hello @F4HZA  o/

I have not tested the Revision S yet - this is the reason that I didn't promote it to the Main branch.

After consulting my digikey and JLCPCB orders, I found that I ordered the parts but never ordered the board - last known good revision is R.

When I get back to São Paulo (I'm in holidays here) I'll check my parts inventory and if all is sound, order a new board revision.

Side note, I found that this BFR193F is... **tiny**. Will be a pain to solder it; the 2SC5086 was already a challenge.

You might also want to check [this reddit thread](https://www.reddit.com/r/rfelectronics/comments/yzsyzt/help_choosing_a_transistor/) where I was suspecting that the BFR193F wasn't a suitable choice (spoiler alert: it seems it was the right choice). 

Expect to have the Rev. S tested by mid March. Will keep this issue open for the time being.

**Comment by rfrht** [comment]
Hello there @F4HZA !

Stopping by to let you know that I ordered the parts and the board this weekend - and will probably have all the ducks lined for testing in January!

**Comment by F4HZA** [comment]
Good morning,
Sorry I moved on to another project, I made myself a panadaptor with an SPF5043Z and the basic schema.
No switch, the FT-991A automatically switches to ground, no filter. I am very happy with it.
I made a PCB with HF connectors, and placed it in the FT-911A since January 2023. I use an SDRPlay the gain is positioned at 1 in the SDR console program.
The important thing is to correctly configure the IF Output in External Radio, something capricious.

Thank you for your project which allowed me to install a panadaptor.


73. F4HZA

**Comment by rfrht** [comment]
Hello, Frederic!
Thanks for sharing the SPF5043Z, I checked its datasheet - and the automatic bias control is definitely a relief.

I built a board design that allows for a SPF5043 for testing, will put that on my future tests. Thank you **very** much for sharing!

So- how's your board working? Do you have any pictures to share?

**Comment by F4HZA** [comment]
Hi,
I didn't take a photo of the mounted card, here is the image of the PCB.
![image](https://github.com/rfrht/FT991A-PAT/assets/62290871/b6576fc8-6905-4dd2-b018-ba95820a6129)

I did it the old fashioned way, copper plate and iron perchloride. The components are quite large and soldering is easy.
I have attached the EAGLE project. 
[Panadapter EAGLE with SPF5043Z - F4HZA .zip](https://github.com/rfrht/FT991A-PAT/files/13447043/Panadapter.EAGLE.with.SPF5043Z.-.F4HZA.zip)
 The card works perfectly, it was installed in January 2023 I have not reopened the FT-991A since. I use it with the SDRPLAY PRO2.
 
A point to review would be the 78L05 which heats up a little. Certainly also the adaptation of the values of the components I do not have the equipment to analyze. Make a professional PCB and resize it for good integration.
I had some problems correctly configuring the SDR CONSOLE software with the "External RADIO" and IF OUTPUT .
73. FAHZA.

**Comment by rfrht** [comment]
Hey there @F4HZA !

I finally built the test board, based on the experimental Revision S. Two major problems:

1. The RF switch is flipped :-P It is disconnected by default - and this is the wrong logic. I have to fix that.
2. I found more or less what you found. While the loss wasn't that much you found, it is a non-neglectible -7 dB. 

I ran a VNA sweep from 10-1000 MHz and found that the board performs reasonably well around 200ish MHz - wich makes me believe that the BFR193 is under heavy impedance mismatch at ~ 70 MHz (the S21 Marker 1).

![s21](https://github.com/rfrht/FT991A-PAT/assets/3044503/6d7ecd62-909a-4b05-8d19-9e0bc1f2fba4)

I'll do some experiences here, and keep this issue open for the time being. I would like to thank you **very much** for the initial test and for reporting this issue!
